src/helpers/serializer.ts

```diff
@@ -0,0 +1,99 @@
+import { Fuego } from '..'
```

Comment:
This one should also probably import directly from the main file.

Reply:
For some reason I thought that was the main file, haha. Fixing it now.